gen/kerberos_renewal_principals

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use File::Basename;
+
+local $::SERVICE_NAME = basename($0);
+local $::PROTOCOL_VERSION = "3.0.0";
+my $SCRIPT_VERSION = "3.0.0";
+
+perunServicesInit::init;
+my $data = perunServicesInit::getHashedHierarchicalData;
+my $DIRECTORY = perunServicesInit::getDirectory;
+
+
+our $A_PRINCIPALS;                    *A_PRINCIPALS =                       \'urn:perun:user:attribute-def:virt:kerberosLogins';
```

Comment:
Service required attribute should be `user:def:kerberosLogins`

Reply:
Switched `virt` to `def`.